pkg/operator/ceph/cluster/mgr/mgr_test.go

```diff
@@ -181,116 +177,52 @@ func TestUpdateServiceSelectors(t *testing.T) {
 	}
 	c := &Cluster{spec: spec, context: ctx, clusterInfo: clusterInfo}
 
-	t.Run("initial active daemon", func(t *testing.T) {
-		activeDaemon := "a"
-		err := c.reconcileServices(activeDaemon)
+	// Make sure we remove the daemon_id label from the selector
+	// of all services with a label "app=rook-ceph-mgr"
+	t.Run("remove daemon_id from mgr services", func(t *testing.T) {
+		svc_1 := corev1.Service{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "svc_1",
```

Comment:
I don't believe underscores can be used in K8s resource names, though it must not matter in the mocked unit tests.
```suggestion
				Name: "svc1",
```